### PR TITLE
till 4

### PR DESCRIPTION
**Pull request by kkenjeres**

### CODE REVIEW

File: src/components/SongList.jsx [hunk 1]
```diff
@@ -3,14 +3,30 @@ import React from 'react'
...
     <>
-      <ul className="space-y-2">
-        {album.tracks.map((track) => (
-          <li key={track.id} className='text-[14px]'><a key={track.id} href={track.spotifyLink} target="_blank" rel="noopener noreferrer" className="text-blue-600 hover:text-blue-800">
-            {track.name}
-          </a></li>
-        ))}
-      </ul>
-    </>
+  <div className="space-y-2">
```
Comment (onemanwenttomow) [inline src/components/SongList.jsx:6]: I feel like it makes more sense to make this a list. Either a <ul> or an <ol>

File: src/components/SongList.jsx [hunk 2]
```diff
@@ -3,14 +3,30 @@ import React from 'react'
...
+    {album.tracks.map((track) => (
+      <a
```
Comment (onemanwenttomow) [inline src/components/SongList.jsx:8]: Which then means that each list item should be wrapped in an <li>

File: src/pages/Search.jsx [hunk 3]
```diff
@@ -1,27 +1,37 @@
...
 
-  const filteredData = data.filter(album => 
+ 
+  useEffect(() => {
+    async function fetchAlbums() {
+      try {
+        const response = await fetch("https://neuefische-spotify-proxy.vercel.app/api/search?artist=${query}");
+        const data = await response.json();
+        setAlbums(data);
+        console.log(data);
```
Comment (onemanwenttomow) [inline src/pages/Search.jsx:20]: Whilst i am a big fan of console logs for development, we should remove them before merging into main.

File: src/pages/Search.jsx [hunk 4]
```diff
@@ -1,27 +1,37 @@
...
   const handleChange = (e) => {
     setSearch(e.target.value);
   }
 
-  const filteredData = data.filter(album => 
+ 
+  useEffect(() => {
+    async function fetchAlbums() {
+      try {
+        const response = await fetch("https://neuefische-spotify-proxy.vercel.app/api/search?artist=${query}");
```
Comment (onemanwenttomow) [inline src/pages/Search.jsx:17]: This is meant to be a dynamic query. So i would expect the use of backticks here. 

The would mean that as the value of query changes, so does that url that we fetch data from. 

i.e. 

``` js
`https://neuefische-spotify-proxy.vercel.app/api/search?artist=${search}`
```

We would need to update the useEffect to run every time `search` changes.

**Review by onemanwenttomow — COMMENTED** [review]
Nice job so far! 

I added a few comments for you to take a look at. 

Also, i see you are using tailwind. When i use it, i like to use it in collaboration with [class sorting with prettier](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier). No requirement to use it, i just wanted to share.

File: src/components/SongList.jsx (hunk 1 above)
Comment (kkenjeres) [reply src/components/SongList.jsx]: import React from "react";

const SongList = ({ album }) => {
  return (
    <>
      <ul className="max-h-[400px] space-y-2 overflow-y-auto p-2">
        {album.tracks.map((track) => (
          <li>
            <a
              key={track.id}
              href={track.spotifyLink}
              target="_blank"
              rel="noopener noreferrer"
              className="block flex justify-between gap-10 rounded-lg bg-[#f0f0f0] p-2 text-black hover:text-blue-600"
            >
              <span className="text-[12px]">{track.track_number}</span>
              <span className="text-[12px]">{track.name}</span>
              <span className="text-[12px]">{track.duration}</span>
            </a>
          </li>
        ))}
      </ul>
    </>
  );
};

export default SongList;

Comment (kkenjeres) [reply src/components/SongList.jsx]: <ul className="max-h-[400px] space-y-2 overflow-y-auto p-2">
        {album.tracks.map((track) => (
          <li>
            <a
              key={track.id}
              href={track.spotifyLink}
              target="_blank"
              rel="noopener noreferrer"
              className="block flex justify-between gap-10 rounded-lg bg-[#f0f0f0] p-2 text-black hover:text-blue-600"
            >
              <span className="text-[12px]">{track.track_number}</span>
              <span className="text-[12px]">{track.name}</span>
              <span className="text-[12px]">{track.duration}</span>
            </a>
          </li>
        ))}
      </ul>

**Pull request merged**